chat/models.tsp

```diff
@@ -454,11 +384,9 @@ model ChatCompletionRequestAssistantMessage {
 
   tool_calls?: ChatCompletionMessageToolCalls;
 
-  /**
-   * Deprecated and replaced by `tool_calls`. The name and arguments of a function that should be
-   * called, as generated by the model.
-   */
-  #deprecated "deprecated"
```

Comment:
Should we keep the #deprecated annotation?

Reply:
Systematically added.